content/models.py

```diff
@@ -3,27 +3,49 @@
 
 
 class OurStory(models.Model):
-    img = models.ImageField(blank=True, null=True)
-    text = models.TextField()
-    color = models.CharField(max_length=20, blank=True)
-    order = models.PositiveIntegerField()
+	img = models.ImageField(upload_to='aboutus', blank=True, null=True)
+	text = models.TextField()
+	color = models.CharField(max_length=20, blank=True)
+	order = models.PositiveIntegerField()
 
 
 
-    def __str__(self):
-        return self.text
+	def __str__(self):
+		return self.text
 
 
 class Ourteam(models.Model):
-    img = models.ImageField(upload_to='uploads')
-    text = models.TextField()
-    us_team = models.CharField(max_length=100)
-    peru_team = models.CharField(max_length=100)
-    board_team = models.CharField(max_length=100)
+	img = models.ImageField(upload_to='aboutus', blank=True, null=True)
+	text = models.TextField()
+	us_team = models.CharField(max_length=100)
+	peru_team = models.CharField(max_length=100)
+	board_team = models.CharField(max_length=100)
 
 
 class OurStoryTitle(models.Model):
-    title = models.TextField()
-
-    def __str__(self):
-        return self.title
+	title = models.TextField()
+
+	def __str__(self):
+		return self.title
+
+class EduProgram(models.Model):
+	title = models.CharField(blank=True, null=True, max_length=100)
+	img = models.ImageField(upload_to='education',blank=True, null=True, help_text="Only upload an image if order = 1")
+	text = models.TextField(blank=True, null=True, help_text="Enter a short description of program.")
+	color = models.CharField(max_length=20, blank=True, help_text="Both 'red' and '#FF0000' are accceptable")
```

Comment:
should prob add in the help text here that the color only works if its not the first/top block